### ISSUE TITLE
RTL Support

### ISSUE THREAD

**Issue by kfirba** [ilyashubin/scrollbooster#22]
Hey!

It seems like this package is incompatible with RTL layouts (scrolling horizontally). Here's a fork of your example with `dir="rtl"` on the HTML element:[ https://codesandbox.io/s/scrollbooster-examples-gfu7d]( https://codesandbox.io/s/scrollbooster-examples-gfu7d)

Do you think this is something you can fix?

Thanks1

**Comment by Mehdi-Hp** [comment]
Yup. I need the same functionality. I'll se if I can contribute.

**Comment by lhaynes96** [comment]
@kfirba @Mehdi-Hp did you find any solutions for this?

**Comment by Mehdi-Hp** [comment]
@ilyashubin Unfortunately not
